quiz/backend/pytest.ini

```diff
@@ -0,0 +1,14 @@
+[pytest]
```

Comment:
Skoro korzystamy z pyproject.toml to konfiguracja do pytesta powinna być tam, a nie tutaj -> usuń proszę ten pli;k https://docs.pytest.org/en/7.2.x/reference/customize.html#pyproject-toml

Reply:
zrobione